composer.json

```diff
@@ -35,7 +35,7 @@
   },
   "autoload-dev": {
     "psr-4": {
-      "Djunehor\\Sms\\Test\\": "tests"
+      "Djunehor\\Sms\\Tests\\": "tests"
```

Comment:
This unnecessary change breaks the app. The correct namespace for tests is `Djunehor\Sms\Test\`

Reply:
Ok. I have been having a little difficult setting up my testbench that's probable why I didn't catch that.

Reply:
Kindly revert back. Once you do, you'll see that the scrutinizer tests will pass

Reply:
alright